coverage_comment/coverage.py

```diff
@@ -82,10 +85,26 @@ class DiffCoverage:
     files: dict[pathlib.Path, FileDiffCoverage]
 
 
-def compute_coverage(num_covered: int, num_total: int) -> decimal.Decimal:
-    if num_total == 0:
+def compute_coverage(
+    num_covered: int,
+    num_total: int,
+    use_branch_coverage: bool,
+    num_branches_covered: int | None,
+    num_branches_total: int | None,
```

Comment:
Rather than conditionally including the branch numbers, you could just arrange to have them default to `0` if there's no branch data. If they're both `0` they won't affect the math, so they can just be added in unconditionally.